components/withSidebarCrossLinks.tsx

```diff
@@ -12,28 +12,39 @@ const WithSidebarCrossLinks: FC<WithCrossLinksProps> = ({ navKey }) => {
 
   const [[, sidebarNavigation]] = getSideNavigation([navKey]);
 
+  if (!sidebarNavigation || !sidebarNavigation.items) {
+    return null; // Return null if sidebar navigation or its items are not available
+  }
+
   const crossLinkItems = sidebarNavigation.items
     .map(([, { items }]) => items.map(([, item]) => item))
     .flat();
 
-  const currentItem = crossLinkItems.findIndex(({ link }) => link === pathname);
+  const validCrossLinkItems = crossLinkItems.filter(({ link }) => link);
```

Comment:
Can you show screenshots of what this is supposed to fix? What is not working at the moment for this? 🤔